src/shared/forms.js

```diff
@@ -0,0 +1,20 @@
+export const messages = {
```

Comment:
Yep, it is at this stage – but the nature of these messages is that they are could be used on more than one form, hence centralising them in `src/shared`.